src/proxy/ITopic.ts

```diff
@@ -48,8 +55,15 @@ export interface ITopic<E> extends DistributedObject {
      *
      * If the given listener already removed, this method does nothing.
      *
+     * This method is for only Reliable Topic.
+     * When this method is called from Topic, it throws {@link HazelcastError}.
+     * Check removeListener method for using with Topic.
+     *
+     * `removeMessageListener` is @deprecated since 5.3. Use {@link removeListener} instead.
+     *
      * @param listenerId listener registration ID
      * @return `true` if registration is removed, `false` otherwise
+     * @throws {HazelcastError} if it is used from Topic
```

Comment:
```suggestion
     * This method is for only Reliable Topic.
     * When this method is called from Topic, it throws {@link HazelcastError}.
     * Check removeListener method for using with Topic.
     *
     * @deprecated `removeMessageListener` is deprecated since 5.3. Use {@link removeListener} instead.
     *
     * @param listenerId listener registration ID
     * @return `true` if registration is removed, `false` otherwise
     * @throws {@link HazelcastError} if it is used from Topic
```

Reply:
Same as above. deprecated done but links are the same.